### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by aaarichter**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "testium-driver-wd",
-  "version": "2.7.0",
+  "version": "2.5.0",
```
Comment (jkrems) [inline package.json]: This looks like an out-of-date base commit..?

Comment (aaarichter) [reply package.json]: thx . rebase mistake. fixing

File: package.json [hunk 2]
```diff
@@ -26,32 +26,32 @@
     }
   },
   "dependencies": {
-    "assertive": "^2.1.1",
-    "debug": "^2.2.0",
+    "assertive": "^3.0.0",
```
Comment (dbushong) [inline package.json]: This driver implicitly exposes some of the assertive API through its `.assert....()` methods - is there anything in the 2 -> 3 breaking change of assertive that would make this a breaking change?

File: package.json [hunk 3]
```diff
@@ -26,32 +26,32 @@
...
+    "debug": "^4.1.1",
     "gofer": "^3.5.8",
-    "lighthouse": "^3.0.1",
-    "lodash": "^4.6.1",
-    "puppeteer": "^1.5.0",
-    "testium-cookie": "^1.0.0",
-    "uuid": "^3.1.0",
-    "wd": "^1.10.1"
+    "lighthouse": "^4.1.0",
```
Comment (dbushong) [inline package.json]: ditto lighthouse major bump

**Comment by dbushong** [comment]
Is there any specific upgrade in here that you need, out of curiosity?

**Comment by jkrems** [comment]
*holds of merging until David's concerns are resolved*

File: package.json (hunk 3 above)
Comment (aaarichter) [reply package.json]: v4.0.0 updates to the new scoring system that was introduced in 2018. If lighthouse is exposed through testium, any deviation to the complex configuration documentation on the lighthouse repo should be avoided.

**Comment by aaarichter** [comment]
> Is there any specific upgrade in here that you need, out of curiosity?

I'm starting to explore Jira LAT-2 and I wanted to have every package at it's latest. esp. lighthouse/puppeteer.

**Comment by aaarichter** [comment]
I have to make an additional change. Some of the accessibility tests don't exist anymore

**Pull request merged**